README.md

```diff
@@ -1,100 +1,222 @@
-# Hubble
+Table of Contents
+=================
 
-An alternate version of Hubblestack which can be run without an existing
-SaltStack infrastructure.
+   * [HUBBLE](#hubble)
+      * [Packaging / Installing](#packaging--installing)
+         * [Installing using setup.py](#installing-using-setuppy)
+         * [Building standalone packages (CentOS)](#building-standalone-packages-centos)
+         * [Building standalone packages (Debian)](#building-standalone-packages-debian)
+         * [Buidling Hubble packages through Dockerfile](#buidling-hubble-packages-through-dockerfile)
+      * [Nova](#nova)
+         * [Usage](#usage)
+         * [Configuration](#configuration)
+      * [Nebula](#nebula)
+         * [Usage](#usage-1)
+         * [Configuration](#configuration-1)
+      * [Pulsar](#pulsar)
+         * [Usage](#usage-2)
+         * [Configuration](#configuration-2)
+            * [Excluding Paths](#excluding-paths)
+            * [Pulsar topfile top.pulsar](#pulsar-topfile-toppulsar)
 
-# Packaging / Installing
+# HUBBLE
 
-## Installing using setup.py
+Hubble is a modular, open-source, security & compliance auditing framework which is built on SaltStack. It is alternate version of Hubblestack which can be run without an existing SaltStack infrastructure. Hubble provides on-demand profile-based auditing, real-time security event notifications, alerting and reporting. It also reports the security information to Splunk. This document describes installation, configuration and general use of Hubble.
 
-```bash
+## Packaging / Installing
+### Installing using setup.py
+```sh
 sudo yum install git -y
 git clone https://github.com/hubblestack/hubble
 cd hubble
 sudo python setup.py install
 ```
+Installs a hubble "binary" into `/usr/bin/`.
 
-Installs a `hubble` "binary" into `/usr/bin/`.
+A config template has been placed in `/etc/hubble/hubble`. Modify it to your specifications and needs. You can do `hubble -h` to see the available options.
 
-## Building standalone packages (CentOS)
+The first two commands you should run to make sure things are set up correctly are `hubble --version` and `hubble test.ping`.
 
-```bash
-sudo yum install git -y
-git clone https://github.com/hubblestack/hubble
-cd hubble/pkg
-./build_rpms.sh  # note the lack of sudo, that is important
-```
-
-Packages will be in the `hubble/pkg/dist/` directory. The only difference
-between the packages is the inclusion of `/etc/init.d/hubble` for el6 and
-the inclusion of a systemd unit file for el7. There's no guarantee of glibc
-compatibility.
-
-## Building standalone packages (Debian)
+### Buidling Hubble packages through Dockerfile
+Dockerfile aims to build the Hubble v2 packages easier. Dockerfiles for the distribution you want to build can be found at the path `hubblestack/hubble/pkg`. For e.g. for centos6 distribution the dockerfile is at the path `hubblestack/hubble/pkg/centos6/` 
```

Comment:
Can you remove the `hubblestack/hubble/` from these paths? Any relative paths assume the root of the repo, so just `pkg/` and `pkg/centos6/` should suffice. You also have an extra "For" at the beginning of the last sentence.